invenio_app_ils/records/loaders/schemas/json/document_request.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""DocumentRequest schema for marshmallow loader."""
+
+from invenio_records_rest.schemas.fields import PersistentIdentifier
+from marshmallow import Schema, fields
+
+
+class DocumentRequestSchemaV1(Schema):
+    """Document Request schema."""
+
+    pid = PersistentIdentifier()
+    state = fields.Str(required=True)
+    patron_pid = fields.Str(required=True)
+    title = fields.Str(required=True)
+    document_pid = fields.Str()
+    authors = fields.Str()
```

Comment:
We might want to have an object or an array here

Reply:
The purpose of keeping it a string was to make it easy for the user since they will be entering this information. Once the request is fulfilled the librarian will create a document based on what the user entered.